src/components/CityListScreen.js

```diff
@@ -35,4 +85,15 @@ const style = StyleSheet.create({
   },
 });
 
-export default CityListScreen;
+const mapStateToProps = state => ({
+  cities: getCities(state),
+});
+
+const mapDispatchToProps = dispatch => ({
+  fetchCurrentWeatherForCityAction: cityName => dispatch(fetchCurrentWeatherForCity(cityName)),
```

Comment:
you might want to look at `bindActionCreators` from redux. It's a nice little helper that makes code a bit prettier - https://redux.js.org/api-reference/bindactioncreators